File: Codeforces/228A.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    ios::sync_with_stdio(0);
    cin.tie(0);

    int n1,n2,n3,n4;
    cin>>n1>>n2>>n3>>n4;

    int output = 0;

    if(n1 == n2 || n1 == n3 || n1 == n4 ) output++;

    if(n2 == n3 || n2 == n4) output++;

    if(n3 == n4) output++;

    cout<<output<<"\n";






    return 0;
}

File: Codeforces/228A.py
n1,n2,n3,n4 = map(int, input().split(" "))

output = 0

if n1 == n2 or n1 == n3 or n1 == n4:
    output+=1
if n2 == n3 or n2 == n4:
    output+=1
if n3 == n4:
    output+=1
    
print(output)


